app/internal/wm/os_js.go

```diff
@@ -583,6 +591,26 @@ func (w *window) windowMode(mode WindowMode) {
 	}
 }
 
+func (w *window) navigationColor(c color.NRGBA) {
+	if c.A != 255 {
+		alpha := float32(c.A) / 255
+		rgba := f32color.LinearFromSRGB(c)
+		rgba.R *= alpha
+		rgba.G *= alpha
+		rgba.B *= alpha
+		rgba.A = 1
+		c = rgba.SRGB()
```

Comment:
I think you can just use f32color.NRGBAToRGBA and throw its alpha channel away. RGBA has alpha pre-multiplied, so the alpha channel can safely be ignored when not blending with the background.

```
rgba := f32color.NRGBAtoRGBA(c)
...
theme.Set("content", fmt.Sprintf("#%08X", []byte{rgba.R, rgba.G, rgba.B, 255}))
```

Please do test, say, white color with alpha = 180 and test that the resulting color has R,G,B = 180.

Reply:
The `f32color.NRGBAToLinearRGBA` returns `180`, when using `{R: 255, G: 255, B: 255, A: 180}`.

Reply:
NRGBAtoRGBA is the correct function, but my example was wrong because I forgot that alpha values are linear. You need to use A=128=0x80. I tested

NRGBAtoRGBA(color.NRGBA{A: 0x80, R: 0xff, G: 0xff, B: 0xff}) = color.RGBA{R:0xbc, G:0xbc, B:0xbc, A:0x80}

R,G,B = 0xbc = 188 is not quite 180, but that's the difference between gamma = 2.2 (R,G,B=188) and gamma = 2.0 (R,G,B=180).

Sorry for the confusion.